Source/AliveLib/MainMenu.cpp

```diff
@@ -1672,21 +1672,268 @@ void MainMenuController::tGame_BackStory_Or_NewGame_Unload_4D1BE0()
+        field_F4_resources.field_0_resources[0] = nullptr;
+        ResourceManager::Reclaim_Memory_49C470(0);
+
+        if (sActiveHero_5C1B68 == spAbe_554D5C)
+        {
+            auto abe = alive_new<Abe>();
+            if (abe)
+            {
+                sActiveHero_5C1B68 = abe->ctor_44AD10(58808, 85, 57, 55);
+            }
+            else
+            {
+                sActiveHero_5C1B68 = 0;
+            }
+        }
+        sActiveHero_5C1B68->field_B8_xpos = FP_FromInteger(0);
+        sActiveHero_5C1B68->field_BC_ypos = FP_FromInteger(0);
+        word_5C1BA0 = 1;
+
+        if (field_208_transition_obj)
+        {
+            field_208_transition_obj->field_6_flags.Set(Options::eDead);
+        }
+        if (field_20C)
+        {
+            field_20C->field_6_flags.Set(Options::eDead);
+        }
+        if (field_210_pUnknown)
+        {
+            field_210_pUnknown->field_6_flags.Set(Options::eDead);
+        }
+        field_6_flags.Set(Options::eDead);
+
+        demoId = word_5C1B9E;
+        if (word_5C1B9C <= 0)
+        {
+            byte_5C1BA2++;
+            if (byte_5C1BA2 > maxDemoId)
+            {
+                byte_5C1BA2 = 1;
+            }
+            demoId = byte_5C1BA2;
+        }
+        char file[32] = {};
+        sprintf(file, "ATTR%04d.SAV", sDemos_5617F0[demoId].field_A_id);
+        ResourceManager::LoadResourceFile_49C170(file, nullptr);
+        BYTE **resource = ResourceManager::GetLoadedResource_49C2A0(1349810254, 0, 1u, 0);
+        sActiveQuicksaveData_BAF7F8 = *(reinterpret_cast<Quicksave*>(*resource));
+        ResourceManager::FreeResource_49C330(resource);
 
+        if (word_5C1B9C)
+        {
+            sActiveQuicksaveData_BAF7F8.field_200_accumulated_obj_count = 1024;
+        }
+        Quicksave_LoadActive_4C9170();
+    }
+    else
+    {
+        field_23C_T80.Set(Flags::eBit18_Loading);
+    }
     return 0;
 }
 
+ALIVE_VAR(1, 0x561538, int, word_561538, 0);
+
+signed int MainMenuController::tDemoSelect_4D0E10(DWORD input)
+{
+    word_5C1BA0 = 0;
+    word_5C1B9C = 0;
+
+    int input_or_field_204 = input;
+    if (field_204_prev_pressed && field_204_prev_pressed == static_cast<int>(sInputObject_5BD4E0.field_0_pads[sCurrentControllerIndex_5C1BBE].field_0_pressed))
+    {
+        field_202_input_hold_down_timer--;
+        if (field_202_input_hold_down_timer == 0)
+        {
+            input_or_field_204 = field_204_prev_pressed;
+            field_202_input_hold_down_timer = 4;
+        }
+    }
+    else
+    {
+        field_202_input_hold_down_timer = 15;
+        field_204_prev_pressed = sInputObject_5BD4E0.field_0_pads[sCurrentControllerIndex_5C1BBE].field_0_pressed;
+    }
+
+    if (input_or_field_204 & 5)
+    {
+        if (field_230_selected_entry_index > 0 && !field_254)
+        {
+            field_230_selected_entry_index--;
+            SFX_Play_46FBA0(0x34u, 35, 400, FP_FromInteger(1));
```

Comment:
The last argument to SFX_Play_46FBA0 is defaulted to 1 AFAIK, so you probably don't need all these.